api/src/repositories/security-repository.ts

```diff
@@ -161,7 +161,7 @@ export class SecurityRepository extends BaseRepository {
   }
 
   /**
-   * Delete persecution rules for a given artifact uuid
+   * Deletes all persecution and hard rules for a given UUID if they exists
```

Comment:
hmm... 😜

Reply:
Also, I guess this should really say, 'deletes all security rules' (we only have persecution rules atm, but I suppose that will eventually change when custom rules, etc, get added.